test/Fail/Issue1952.err

```diff
@@ -99,75 +99,78 @@ Failed to solve the following constraints:
     D : Structure
     C : Structure
     (stuck)
-  (_r_173 Structure.⟶
-   (_r_173 Structure.× (_r_173 Structure.× _X_174) _X_174) _X_174)
+  _r_173 .Structure._⟶_
+  (_r_173 .Structure._×_ (_r_173 .Structure._×_ _X_174 _X_174)
+   _X_174)
```

Comment:
Here the printing became accidentially better, but the underlying problem is #632.